src/main/java/org/truffleruby/RubyLanguage.java

```diff
@@ -107,6 +107,7 @@ protected CallTarget parse(ParsingRequest request) throws Exception {
         return Truffle.getRuntime().createCallTarget(new LazyRubyRootNode(this, null, null, request.getSource(), request.getArgumentNames()));
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Do we have a way to audit these? Should you add a TODO here?

Reply:
I'll open an issue for them.